docs/diagrams/AddPatientSequenceDiagram.puml

```diff
@@ -1,11 +1,14 @@
 @startuml
 
+!include style.puml
+skinparam ArrowFormatStyle plain
+
 actor User
-participant MediTask
-participant Parser
-participant AddParser
-participant AddPatientCommand
-participant hospital
+participant ":MediTask" as MediTask MAIN_COLOR
+participant ":Parser" as Parser PARSER_COLOR
+participant ":AddParser" as AddParser PARSER_COLOR
+participant ":AddPatientCommand" as AddPatientCommand COMMAND_COLOR
+participant ":Hospital" as hospital DATA_COLOR
```

Comment:
Good job standardising the color for our diagrams - Looks better and more unified.